### ISSUE TITLE
On/Off switches are not accessible to screen readers

### ISSUE THREAD

**Issue by WilHall** [thoughtbot/constable#723]
The On/Off switches are not accessible to screen reader users, so tabbing through the form skips right over them (tested using VoiceOver on OS X):
<img width="400" alt="Image 2019-09-12 at 8 10 43 PM" src="https://user-images.githubusercontent.com/342826/64829203-761f8300-d599-11e9-881d-cd155aa857bc.png">

**Comment by ericwbailey** [comment]
Here's a battle-tested switch pattern I quite enjoy: https://scottaohara.github.io/a11y_styled_form_controls/src/checkbox--switch/

**Comment by WilHall** [comment]
@ericwbailey thanks so much for that! That example was super informative.
